docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+
+  db:
+    image: postgres:13-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=postgres
+    ports:
+      - "5432:5432"
+
+  bot:
+    build: .
```

Comment:
you can build here, but it is better to build your docker image every time you merge smth to master. You can do that inside continuous integration process (CI). 
https://github.com/marketplace/actions/build-and-push-docker-images
Here you can pull the latest image you built.

Reply:
I did.